tests/e2e/support/commands.js

```diff
@@ -1166,8 +1166,11 @@ Cypress.Commands.add('loadGameFixture', (pNum, fixture) => {
     return cy
       .window()
       .its('cuttle.gameStore.id')
-      .then((gameId) => {
-        return cy.makeSocketRequest(`game/${gameId}`, 'loadFixtureGameState', fixture);
+      .then(async (gameId) => {
+        await cy.makeSocketRequest(`game/${gameId}`, 'loadFixtureGameState', fixture);
+        const playerHandLength = pNum === 0 ? fixture.p0Hand.length : fixture.p1Hand.length;
+        cy.get('[data-player-hand-card]').should('have.length', playerHandLength);
```

Comment:
The gamestate api case here was missing a DOM assertion to confirm that the request finished processing before ending the command. This way we can immediately call things like `cy.playOneOffOpponent()` because the cards will be in the players' hands before the next line of the test runs.